packages/frameworks/src/frameworks.ts

```diff
@@ -1581,6 +1581,59 @@ export const frameworks = [
     buildCommand: 'vite build',
     getOutputDirName: async () => 'dist',
   },
+  {
+    name: 'Astro',
+    slug: 'astro',
+    demo: 'https://astro.examples.vercel.com',
+    logo: 'https://raw.githubusercontent.com/vercel/vercel/main/packages/frameworks/logos/astro.svg',
+    tagline:
+      'Astro is a next-generation static site generator that mixes lightning-fast performance with modern developer experience.',
+    description: 'An Astro site, using the default starter kit.',
+    website: 'https://astro.build',
+    envPrefix: 'SNOWPACK_PUBLIC_',
+    detectors: {
+      every: [
+        {
+          path: 'package.json',
+          matchContent:
+            '"(dev)?(d|D)ependencies":\\s*{[^}]*"astro":\\s*".+?"[^}]*}',
+        },
+      ],
+    },
+    settings: {
+      installCommand: {
+        placeholder: '`yarn install` or `npm install`',
+      },
+      buildCommand: {
+        placeholder: '`npm run build` or `astro build`',
+      },
+      devCommand: {
+        value: 'astro dev',
+      },
+      outputDirectory: {
+        value: 'dist',
+      },
+    },
+    dependency: 'astro',
+    devCommand: 'astro dev',
+    buildCommand: 'astro build',
+    getOutputDirName: async () => 'dist',
+    cachePattern: '.snowpack/**',
+    defaultRoutes: [
+      {
+        src: '^/_astro/(.*)\\.css$',
```

Comment:
@fredkschott can you verify that this will be okay? I believe `_astro`'s `.css` files are all hashed but I'm definitely not a power user.

Reply:
This looks fine for now. JS files are bundled but filenames aren't hashed yet, they will be by v1.0.